src/sentry/api/endpoints/organization_events_trends.py

```diff
@@ -73,8 +83,18 @@ def get(self, request, organization):
 
         count_column = self.trend_columns.get("count_range")
         percentage_column = self.trend_columns["percentage"]
+        variance_column = self.trend_columns["variance_range"]
+        avg_column = self.trend_columns["avg"]
         selected_columns = request.GET.getlist("field")[:]
+        t_score = self.trend_columns["t_score"]["format"].format(
+            avg=avg_column["alias"], variance=variance_column["alias"], count=count_column["alias"],
+        )
+        degrees_of_freedom = self.trend_columns["degrees_of_freedom"]["format"].format(
+            variance=variance_column["alias"], count=count_column["alias"]
+        )
         query = request.GET.get("query")
+        query = query.replace("t_score()", t_score)
+        query = query.replace("degrees_of_freedom()", degrees_of_freedom)
```

Comment:
Are these meant to be search filters in the search bar?

Reply:
Just a nit but you can move these constants up into the dict you declared above as `input_format` or something.

Reply:
:+1: Yeah definitely, planning to do something like this so that these can be used in the filter bar without having to pass all the arguments. (Different, non-spike non-staging-only PR though)